webpack/webpack-utils/build-filepath-lookup.js

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable */
+
+const Path = require('path');
+const fs = require('fs');
+
+function buildFilepathLookup(path, staticUrlPrefix) {
+    if (!fs.existsSync(path)) {
+        return;
+    }
+
+    const prefix = path.match(/[^/]+$/);
+    const staticUrl = staticUrlPrefix ? staticUrlPrefix : "";
+
+    const getFileList = function (dirPath) {
+        return fs.readdirSync(dirPath, { withFileTypes: true }).reduce((fileList,entries) => {
+            const childPath = Path.join(dirPath, entries.name);
+
+            if (entries.isDirectory()) {
+                fileList.push(...getFileList(childPath, fileList));
+            } else
+            {
+                fileList.push(childPath);
+            }
+            return fileList;
+            }, []);
+        };
+
+    return getFileList(path).reduce((lookup, file) => {
+        const extension = file.match(/[^.]+$/).toString();
+        const extensionReplacementRegex = new RegExp(`\\.${extension}$`);
```

Comment:
Now or later we should rerun updateproject to get the fix for dotfiles.

Reply:
done!